pmd-java/src/test/java/net/sourceforge/pmd/lang/java/rule/codestyle/UseDiamondOperator.java

```diff
@@ -0,0 +1,7 @@
+package net.sourceforge.pmd.lang.java.rule.codestyle;
```

Comment:
This file needs the following  license header - that why the [build is failing](https://travis-ci.org/pmd/pmd/jobs/471600654#L3167):

```java
/**
 * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
 */
```